components/nimbus/src/lib.rs

```diff
@@ -36,22 +37,24 @@ use once_cell::sync::OnceCell;
 use persistence::{Database, StoreId, Writer};
 use serde_derive::*;
 use serde_json::{Map, Value};
-use std::collections::HashSet;
 use std::path::PathBuf;
 use std::sync::Mutex;
+use std::{collections::HashSet, str::FromStr};
 use updating::{read_and_remove_pending_experiments, write_pending_experiments};
 use uuid::Uuid;
 
 const DEFAULT_TOTAL_BUCKETS: u32 = 10000;
 const DB_KEY_NIMBUS_ID: &str = "nimbus-id";
-
+pub const DB_KEY_INSTALLATION_DATE: &str = "installation-date";
```

Comment:
I persist the installation date in the `Meta` store, so we don't have to go looking at the directory every single run. Happy to take alternatives/ideas!